package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-makina",
-  "version": "0.1.0",
+  "version": "0.3.0",
```

Comment:
Not very important but why do you jump from 0.1 to 0.3 ?

Reply:
The [current change](359ce534b0c9fe922eacb6cb5d3eab397674c1ec) is from `0.2.0` to `0.3.0`, but the merge request contains also a [previous version bump](74d8cfd47f55e6e5d390152aa813dc33dfc5697f) from `0.1.0` to `0.2.0` so the full changeset of the merge request display the overall version bump.

Reply:
Fine